.travis.yml

```diff
@@ -8,11 +8,22 @@ php:
   - 7.0
   - hhvm
 
+matrix:
+    include:
+        - env: SYMFONY_VERSION=2.3.*
+        - env: SYMFONY_VERSION=2.7.*
+        - env: SYMFONY_VERSION=2.8.*
+        - env: SYMFONY_VERSION=3.0.*
```

Comment:
A minor modification is needed in order to travis to intrepid this correctly. 

See https://github.com/php-http/HttplugBundle/blob/master/.travis.yml#L18-L22